### PR TITLE
adding support for bcc mysqldqslower tool

### PR DESCRIPTION
**Pull request by pyeole**
By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: mod.d/bccmysqldqslower.yaml [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+      exit 0
+  }
+  trap error_trap ERR
+
+   # read-in shared function
+  source functions.bash
+
+   echo "I will collect mysqld_qslower output from this $EC2RL_DISTRO box $times times."
+
+   timeout --preserve-status $period mysqld_qslower `pgrep -n mysqld` $threshold
```
Comment (Drudenhaus) [inline mod.d/bccmysqldqslower.yaml]: Did you intend to do this $times times in a loop?

Comment (pyeole) [reply mod.d/bccmysqldqslower.yaml]: No. That was unintentional. $times is not required here. Added an updated commit.

Comment (gregbdunn) [reply mod.d/bccmysqldqslower.yaml]: You'll want to update lines 46 and 56 to remove times then :)

File: mod.d/bccmysqldqslower.yaml [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+   exit 0
+constraint:
+  requires_ec2: !!str False
+  domain: !!str net
+  class: !!str performance
+  distro: !!str alami alami2 ubuntu rhel suse
+  required: !!str period times threshold
```
Comment (Drudenhaus) [inline mod.d/bccmysqldqslower.yaml]: Need to remove 'times' here too.

Comment (pyeole) [reply mod.d/bccmysqldqslower.yaml]: Updated CHANGELOG and edited the module to remove "times" requirement

Comment (gregbdunn) [reply mod.d/bccmysqldqslower.yaml]: Looks good!

**Pull request merged**